packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -95,19 +76,38 @@ const configurations = {
 
 const { keys } = Object;
 const Usage = `agoric run replace-electorate-core.js ${keys(configurations).join(' | ')}`;
-export default async (homeP, endowments) => {
-  const { scriptArgs } = endowments;
-  const variant = scriptArgs?.[0];
-  const config = configurations[variant];
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }, opts) => {
+  const config = configurations[opts.variant];
   if (!config) {
     console.error(Usage);
     process.exit(1);
   }
-  console.log('replace-committee', scriptArgs, config);
+  return harden({
+    sourceSpec: '@agoric/inter-protocol/src/proposals/replaceElectorate.js',
+    getManifestCall: [
+      getManifestForReplaceAllElectorates.name,
+      {
+        ...config,
+        economicCommitteeRef: publishRef(
+          install(
+            '@agoric/governance/src/committee.js',
+            '../bundles/bundle-committee.js',
+          ),
+        ),
+      },
+    ],
+  });
+};
+
+export default async (homeP, endowments) => {
+  const { scriptArgs } = endowments;
+  const variant = scriptArgs?.[0];
+  console.log('replace-committee', scriptArgs, variant);
 
   const { writeCoreEval } = await makeHelpers(homeP, endowments);
 
   await writeCoreEval(`replace-committee-${variant}`, (utils, opts) =>
-    defaultProposalBuilder(utils, { ...opts, ...config }),
+    defaultProposalBuilder(utils, { ...opts, variant }),
```

Comment:
I would expect a corresponding change to  `getManifestForReplaceAllElectorates`, but I don't see one. How does this work?

Reply:
do we need an update for `getManifestForReplaceAllElectorates `?
the only change made here is to move the config variable from the default exported function to the `defaultProposalBuilder` function. It shouldn't affect `getManifestForReplaceAllElectorates` at all since the values passed to it are the same